core/test/financial-templates/Liquidatable.js

```diff
@@ -1215,4 +1216,335 @@ contract("Liquidatable", function(accounts) {
+        await USDCLiquidationContract.withdrawLiquidation(liquidationParams.liquidationId, sponsor, { from: sponsor });
+        const sponsorUSDCBalanceAfter = await collateralToken.balanceOf(sponsor);
+
+        // Expected Sponsor payment => remaining collateral (locked collateral - TRV) + sponsor reward
+        const USDCExpectedPayment = USDCAmountOfCollateral.sub(USDCSettlementTRV).add(USDCSponsorDisputeReward);
+        assert.equal(sponsorUSDCBalanceAfter.sub(sponsorUSDCBalanceBefore).toString(), USDCExpectedPayment.toString());
+
+        // Sponsor should not be able to call again
+        assert(
+          await didContractThrow(
+            USDCLiquidationContract.withdrawLiquidation(liquidationParams.liquidationId, sponsor, { from: sponsor })
+          )
+        );
+      });
+      it("Liquidator calls", async () => {
+        const liquidatorUSDCBalanceBefore = await collateralToken.balanceOf(liquidator);
+        await USDCLiquidationContract.withdrawLiquidation(liquidationParams.liquidationId, sponsor, {
+          from: liquidator
+        });
+        const liquidatorUSDCBalanceAfter = await collateralToken.balanceOf(liquidator);
+
+        // Expected Liquidator payment => TRV - dispute reward - sponsor reward
+        const expectedPayment = USDCSettlementTRV.sub(USDTDisputerDisputeReward).sub(USDCSponsorDisputeReward);
+        assert.equal(
+          liquidatorUSDCBalanceAfter.sub(liquidatorUSDCBalanceBefore).toString(),
+          expectedPayment.toString()
+        );
+
+        // Liquidator should not be able to call again
+        assert(
+          await didContractThrow(
+            USDCLiquidationContract.withdrawLiquidation(liquidationParams.liquidationId, sponsor, { from: liquidator })
+          )
+        );
+      });
+      it("Disputer calls", async () => {
+        const disputerUSDCBalanceBefore = await collateralToken.balanceOf(disputer);
+        await USDCLiquidationContract.withdrawLiquidation(liquidationParams.liquidationId, sponsor, { from: disputer });
+        const disputerUSDCBalanceAfter = await collateralToken.balanceOf(disputer);
+
+        // Expected Disputer payment => disputer reward + dispute bond
+        const expectedPayment = USDTDisputerDisputeReward.add(USDCDisputeBond);
+        assert.equal(disputerUSDCBalanceAfter.sub(disputerUSDCBalanceBefore).toString(), expectedPayment.toString());
+
+        // Disputer should not be able to call again
+        assert(
+          await didContractThrow(
+            USDCLiquidationContract.withdrawLiquidation(liquidationParams.liquidationId, sponsor, { from: disputer })
+          )
+        );
+      });
+      it("Rando calls", async () => {
+        assert(
+          await didContractThrow(
+            USDCLiquidationContract.withdrawLiquidation(liquidationParams.liquidationId, sponsor, { from: rando })
+          )
+        );
+      });
+      it("Withdraw still succeeds even if Liquidation has expired", async () => {
```

Comment:
nit: similar to above, is this test relevant?

Reply:
ok I removed all tests that do not directly test raw collateral changing which would be affected by decimalization

Reply:
Sorry, should've probably worded my comments better. I just wanted to better understand how you picked the tests to include vs not include -- wasn't implying that you should necessarily take them out.